src/Handler/Bucket/Drop/RevokeBucketAccessFromReadOnlyRoleHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\BigQuery\Handler\Bucket\Drop;
+
+use Google\Protobuf\Internal\Message;
+use Keboola\StorageDriver\BigQuery\GCPClientManager;
+use Keboola\StorageDriver\BigQuery\NameGenerator;
+use Keboola\StorageDriver\Command\Bucket\RevokeBucketAccessFromReadOnlyRoleCommand;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+
+final class RevokeBucketAccessFromReadOnlyRoleHandler implements DriverCommandHandlerInterface
+{
+    public GCPClientManager $clientManager;
+
+    public function __construct(GCPClientManager $clientManager)
+    {
+        $this->clientManager = $clientManager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     */
+    public function __invoke(
+        Message $credentials, // project credentials
+        Message $command,
+        array $features,
+        Message $runtimeOptions,
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof RevokeBucketAccessFromReadOnlyRoleCommand);
```

Comment:
Tu som vlastne zbytocne pridaval branchId do commonu v dalsom PR to zmazem, je to rovnake ako delete bucket v podstate, ale nechal som to ako zvlast handler nech dodrzujem "interface"